client/components/beer-listing/beer-listing.jsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import classNames from 'classnames/bind';
 import styles from './beer-listing.scss';
+import { range } from 'ramda'
 
 const css = classNames.bind(styles);
 
 export default function BeerListing({
-  beerType, volume, abv, price, location, personName, personId, rating, description,
+  beerType, volume, abv, price, location, personName, personId, rating, showRating = true, description,
```

Comment:
when using boolean props, it's usually better to default them to `false`. that way we can enable the flag when rendering the component just by writing `<BeerListing showRating />` without any assignment to the prop itself.

Reply:
Good point